docs/user/README.md

```diff
@@ -18,6 +18,13 @@ By default, the cluster will be given the name `kind-1`. "1," here, is the
 default context name. 
 Use the `--name` flag to assign the cluster a different context name.
 
+**Note**: If you are running `kind` of MacOS or Windows then it is recommended
+that you have at least 4GB of RAM and disk space (these are estimates for a
```

Comment:
We should mention or link to how to do this. Many users have never needed to touch this.

Reply:
that's a good point. Added it just below. Tell me what you think